index.rst

```diff
@@ -44,18 +44,589 @@
+
+- M1M3
+- M2
+- M2 hexapod
+- Camera hexapod
+- Camera rotator
+- ComCam (during commissioning)
+- LSSTCam (during commissioning and operations)
+
+Over the years, each component has been under development under their own coordinate system (CS).
+Sometimes the situation can be even more complicated - A single vendor may be using multiple CS for different sub-components (for example, mechanical versus optical). There are also inconsistencies between various engineering models and drawings.
+
+If we include simulations and control algorithm work, there are additional CS that are relevant to the AOS,
+
+- Zemax - as defined in `the project official optical model <https://confluence.lsstcorp.org/display/SYSENG/As-built+optical+model>`__ (This is used to calculate the optical sensitivity matrix)
+- PhoSim :cite:`2015ApJS..218...14P` (This is used for closed-loop simulations)
+
+In order for the AOS to work as one system, a clean understanding of these CS and how they fit together is required. This is the major objective of this technical note. We will also discuss how to transform coordinates and operations between various CS, and more importantly, how to transform everything into a common CS - the Optical CS.
+
+.. note::
+
+   Document-18499 :cite:`Document-18499` defines a set of CS and the transformations between them. However, it doesn't go into all the levels of details that are sufficient for understanding the AOS operations.
+
+.. _section-ocs:
+
+#############################
+The Optical Coordinate System
+#############################
+
+Our AOS system-wide CS is the Optical Coordinate System (OCS). The defition of the OCS is the same as defined in LTS-136 :cite:`LTS-136`, Section 2.2. In summary -
+
+- The origin of the OCS is at the theoretical vertex of M1.
+- +z axis points along the optical axis, toward the sky
+- +x axis goes in parallel with the elevation axis. When look from the sky, +x points to the right.
+- +y axis can then be determined using the right hand rule. At horizon pointing, +y goes up.
+- The azimuth angle is zero when the telescope points toward northern horizon. Looking from the sky, it increase clockwise.
+- The zenith angle is zero when the telescope points at zenith. It increase to 90 degrees for horizon pointing.
+
+.. figure:: /_static/ocs.png
+   :name: fig-ocs
+   :target: ../_images/ocs.png
+   :alt: The Optical Coordinate System (OCS)
+
+Optical Coordinate System (OCS). Left: view from the sky; Right: side view from the -X axis.
+
+The idea is that the OCS is fixed to M1, which defines the reference for the optical system.
+Relative to the ground, the OCS rotates with the azimuth angle and the zenith angle.
+An easy way to identify the +x/+y/+z while looking at a drawing or the actual hardware is that, the walkway on the TMA, above M1M3, is in the +y direction. Otherwise when we go toward horizon it is going to bump into the ground.
```

Comment:
I do not understand this sentence at all. Please help to explain more. Thanks!

Reply:
will add annotation to the figure to point out walkway.